### ISSUE TITLE
PaddleOCR Multi-Container on Single Instance(PC)

### ISSUE THREAD

**Issue by Luyin** [PaddlePaddle/PaddleOCR#6173]
First of all, thank you for developing good software.

I made OCR API using paddleOCR and Python Fast API web framework.

After making this OCR API with Docker, the server was configured so that multiple requests could be processed on one PC (AWS, EC2) by running multiple containers.

However, despite running multiple containers, inference can be performed on one PC at a time.

Are there any options that need to be set separately to handle multiple requests?

```python
    paddle_ch = PaddleOCR(
            show_log=False,
            lang="ch",
            det_db_thresh=0.1,
            det_db_box_thresh=0.1,
            use_mp=True,
            total_process_num=process_count,
            # use_angle_cls=True,
            cls_model_dir="whl/cls/ch_ppocr_mobile_v2.0_cls_infer",
            det_model_dir="whl/det/ch/ch_ppocr_server_v2.0_det_infer",
            rec_model_dir="whl/rec/ch/ch_ppocr_server_v2.0_rec_infer"
    )
```

**Comment by WenmuZhou** [comment]
paddleocr does not impose any restrictions on multi-process

**Comment by Luyin** [comment]
@WenmuZhou 

Thank you for update paddleOCR v2.5

this bug is fixed in v2.5 !!!

**Comment by Luyin** [comment]
@WenmuZhou 

I found reason.

text detector is very slow when I use default value of ```cpu_threads=10``` in Multi-Container

after change ```cpu_threads=1``` It works fine.

Are there any reason why you set cpu_threads to 10 as default value?

and I don't get why the text detector gets slower as I increase the number of threads.
